engineyard/ey-cookbooks-stable-v5
# Resque

## Description of your patch

Port the resque recipe to the v5 stack.

## Recommended Release Notes

Adds the resque recipe to the v5 stack.

## Estimated risk

Low. The main chef recipes are not changed.

## Components involved

Resque custom chef recipe.

## Description of testing done

Add the resque, redis, and custom-resque recipes to cookbooks directory on the local machine. Upload the recipes and click Apply.

## QA Instructions
1. Boot a cluster environment on stable stack using the rails_activejob_example application. Include a utility instance named "resque".
2. Ensure chef runs to completion successfully and app deploys and runs as expected
3. Deploy the "resque" branch of the application.
4. Confirm unicorn or passenger is running.
5. Upgrade to target QA stack.
6. Upload the custom chef recipes with custom-resque and custom-redis. Use examples/resque.
7. Click Apply.
8. Verify that resque is running with "monit summary".
9. Add 100 jobs. On the resque instance, run `cd /data/rails_activejob_example/current; ECHO_JOB_COUNT=100 bundle exec rake echo:generate`
10. Verify that the jobs were processed. On the resque instance, run `wc -l /data/rails_activejob_example/current/tmp/echo.output` should show 100.
